pkg/util/stmtsummary/v2/stmtsummary.go

```diff
@@ -467,7 +467,7 @@ type stmtKey struct {
 // `prevSQL` is included in the key To distinguish different transactions.
 func (k *stmtKey) Hash() []byte {
 	if len(k.hash) == 0 {
-		k.hash = make([]byte, 0, len(k.schemaName)+len(k.digest)+len(k.prevDigest)+len(k.planDigest))
+		k.hash = make([]byte, 0, len(k.schemaName)+len(k.digest)+len(k.prevDigest)+len(k.planDigest)+len(k.resourceGroupName))
```

Comment:
Why the `k.resourceGroupName` is added here?

Reply:
Because `k.resourceGroupName` is the part of key. There will be one additional allocation and mem copy if we do not count it.